Formula/chunkwm.rb

```diff
@@ -0,0 +1,108 @@
+class Chunkwm < Formula
+  desc "Tiling window manager for macOS based on plugin architecture"
+  homepage "https://github.com/koekeishiya/chunkwm"
+  url "https://github.com/koekeishiya/chunkwm/archive/v0.2.28.tar.gz"
+  sha256 "917c6aa09521ff4b7463f731192eea6be6313f0fa4e585f63f85603d0993cf92"
+
+  head do
```

Comment:
Yeah I got the intention. But you don't need specific `stable do` or `head do` unless you need to do specific things within those blocks, like for example if you needed the autotools for generating a `configure` script only when building from `HEAD`. Simply the `head` line with the HEAD URL suffices otherwise.